WindowsFormsApp15/Form2.cs

```diff
@@ -1 +1,109 @@
-﻿
\ No newline at end of file
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using System.IO;
+using System.Text.Json;
+
+namespace WindowsFormsApp15
+{
+    
+    //pls
+
+    public class Comment
+    {
+        private DateTime date;
+        public class User
+        {
+            private string username; 
+            private Image userpic;
+
+            public User(string username, Image userpic)
+            {
+                this.username = username;
+                this.userpic = userpic;
+            }
+        }
+        private string text;
+
+        public Comment(DateTime date, string text)
+        {
+            this.date = date;
+            this.text = text;
+        }
+    }
```

Comment:
Так вот, это должно выглядеть как-то так

```suggestion
public class User
        {
            private string username; 
            private Image userpic;

            public User(string username, Image userpic)
            {
                this.username = username;
                this.userpic = userpic;
            }
        }
    public class Comment
    {
        private DateTime date;
        private string text;
        private User user;

        public Comment(DateTime date, string text, User user)
        {
            this.date = date;
            this.text = text;
            this.user = user;
        }
    }
```